asv_bench/benchmarks/utils.py

```diff
@@ -90,7 +98,20 @@
 }
 
 
-def translator_groupby_ngroups(groupby_ngroups, shape):
+def translator_groupby_ngroups(groupby_ngroups: Union[str, int], shape: tuple) -> int:
+    """
+    Translate a string representation of the number of groups, into a number.
+
+    Parameters
+    ----------
+    groupby_ngroups: str or int
```

Comment:
I prefer using indent between a parameter name and colon as it is presented in numpydoc guide :)

Reply:
@YarShev it should be a quick fix.

Do our doc tests not properly check the numpydoc standard?

Reply:
It seems that `pydocstyle --convention=numpy` doesn't check it. Although, the guide says the following: `The colon must be preceded by a space, or omitted if the type is absent.`

Reply:
Should be "groupby_ngroups : str or int" ie space before :